### ISSUE TITLE
Exception when using a domain

### ISSUE THREAD

**Issue by sylvainlb** [read-write-web/rww-play#152]
When using a domain, RwwPlay fails with a timeout exception unless a http.domain parameter is provided.
RwwPlay should accept requests wherever they're coming if not specified otherwise.

**Comment by reederz** [comment]
I thought that http.domain was used for url generation internally rather than as a security measure.

**Comment by sylvainlb** [comment]
Absolutely. The bug is happening when the parameter is not provided and the host requested is not in the interfaces.

**Comment by bblfish** [comment]
yes, so that means that the server should function with relative URI support. Relative URIs should be always be directed by the server to the local actor. This is then another reason for supporting relative URIs, and even relative Graphs: the server may not know it's own name. So a use case for https://github.com/w3c/banana-rdf/issues/41
